### ISSUE TITLE
Crash without .NET 3.5 framework

### ISSUE THREAD

**Issue by albertowd** [yuzu-emu/yuzu#4024]
On a new windows installation, only the VC++ Redis 2019 will make the emulator crash before the window opening.

The emulator only opened after I've installed the .NET 3.5 framework on my Window 10 Pro.

As this feature does not install on a regular Windows 10 Pro installation, I think it's valid to add on the emulators page this requirement too.

**Comment by auranticus** [comment]
Nope, on my PC it still works without .NET 3.5. VC++ is still a hard requirement.

**Comment by MerryMage** [comment]
.NET should not be a systrem requirement.

**Comment by albertowd** [comment]
I'll assembly a new PC this week and test other possibilities, but the emulator did not opened only with the VC++ redis installed on my Windows 10 2004 x64.

It only stopped crashing before load after I've installed Steam and Doom on my system. The Steam client could have installed another tools that I didn't see while installing the Doom game.

**Comment by auranticus** [comment]
Didn’t Steam tell you what dependencies it is installing for the game (Doom in your instance) before opening?

**Comment by Morph1984** [comment]
Net Framework 3.5 should already be included in all Windows installations.

![image](https://user-images.githubusercontent.com/39850852/83528620-53325780-a4b7-11ea-9e55-6645457a0e99.png)

**Comment by albertowd** [comment]
> ink that you should try install _both_ x86 and x64 o

I've tried install both versions too, but without success. When I build a new PC I'll try that again. If continue to crash I'll paste my logs here, but for what I remember it's an exception in some ntdll file.

**Comment by auranticus** [comment]
Quick search on Google about ntdll file thing, and I can find that it's related to NT kernel stuff. I  think your Windows install is corrupted or something. (If this's not true, you can send the screenshot of that error)
And again, .NET unrelated \*shrug\*

**Comment by albertowd** [comment]
Thanks, I'll try to do that next time. ^^
